docs/src/_getting-started/part1.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Tutorial Part 1 - Extract Data
+description: Part 1 - If you're ready to get started with Meltano and run an EL[T] pipeline with a data source and destination of your choosing, you've come to the right place!
+layout: doc
+weight: 3
+---
+Let’s learn by example.
+
+Throughout this tutorial, we’ll walk you through the creation of a end-to-end modern E(t)LT stack.
+
+We're going to take data from a "source", namely GitLab, and extract a list of projects, the authors, messages and so on.
+To test that this part works, we will dump the data into JSON files.
+In Part 2, we will then place this data into a PostgreSQL database.
+
+We'll assume you have [Meltano installed](/getting-started/installation) already. You can tell Meltano is installed and which version by running ``` meltano version```
+
+![Meltano Version](images/part1/gif_meltano_version.gif)
+
+
+This tutorial is written using meltano >= v2.0.0.
+
+If you don't have a GitLab account to follow along, you could either exchange the commands for a differe tap, like GitHub or PostgreSQL, or you create a free GitLab account. You will also need a [personal access token to your GitLab account](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html) which you can create under "Edit Profile > "Access Tokens" > .
+
+<div class="notification is-success">
+    <p>If you're having trouble throughout this tutorial, you can always head over to the <a href="https://meltano.com/slack">Slack channel</a> to get help.</p>
+</div>
+
+## Create Your Meltano Project
+Step 1 is to create a new [Meltano project](/concepts/project) that (among other things)
+will hold the [plugins](/concepts/plugins) that implement the details of our E(t)LT pipeline.
+
+
+1. Navigate to the directory that you'd like to hold your Meltano projects.
+
+2. Initialize a new project in a directory of your choosing using [`meltano init`](/reference/command-line-interface#init):
+
+   ```bash
+   meltano init my-meltano-project
+   ```
+
+![Meltano Init](images/part1/gif_meltano_init.gif)
+
+   This action will create a new directory with, among other things, your [`meltano.yml` project file](/concepts/project#meltano-yml-project-file). Your file will look something like this:
+
+   ```yml
+   version: 1
+   default_environment: dev
+   project_id: u3u5u4ueudufhfh4h4h5ddkjh
```

Comment:
```suggestion
   project_id: <unique-GUID>
```

It'll default to a true unique GUID so I'd recommend either having a guid there are just indicating that it can be a guid.